### DISCUSSION TITLE
Add Apparent Temperature to Weather & Forecast Entity

### DISCUSSION THREAD

**Discussion by brenank** [home-assistant/architecture#785] [Entity Models] [answered]
Would propose to add Apparent Temperature (feels-like temperature) as a first-class attribute on the weather & forecast entities.  

Apparent Temperature could be supported by NWS integration for both Weather & Forecast.  Certain weather integrations may not provide this data, so it would be an optional attribute.

https://developers.home-assistant.io/docs/core/entity/weather/

**Comment by brenank** [comment]
Related PR: https://github.com/home-assistant/core/pull/76418

**Comment by frenck** [comment, marked as answer]
We have discussed this architectural proposal and are in agreement this property can be added to both the weather entity and the forecast model of the weather entity.

../Frenck

**Comment by gjohansson-ST** [comment]
PR opened: https://github.com/home-assistant/core/pull/95070

**Reply by frenck** [reply]
Merged, so this architectural issue can be closed.
